README.md

```diff
@@ -14,75 +20,66 @@
-**High performance self-hosted photo and video backup solution.**
-
-![](https://media.giphy.com/media/y8ZeaAigGmNvlSoKhU/giphy.gif)
+# Features 
 
-Loading ~4000 images/videos
+> ⚠️ WARNING: **NOT READY FOR PRODUCTION! DO NOT USE TO STORE YOUR ASSETS**. This project is under heavy development, there will be continuous functions, features and api changes.
 
-## Screenshots
+|  | Mobile | Web |
+| - | - | - | 
+| ☁️ Upload and view videos and photos | Yes | Yes 
+| 🔄 Auto backup when the app is opened | Yes | N/A
+| ☑️ Selective album(s) for backup | Yes | N/A
+| ⬇️ Download photos and videos to local device | Yes | Yes
+| 👪 Multi-user support | Yes | Yes
+| 🖼️ Album | Yes | Yes
+| 🤝 Shared Albums | Yes | Yes
+| 🚀 Quick navigation with draggable scrollbar | Yes | Yes
+| 🗃️ Support RAW (HEIC, HEIF, DNG, Apple ProRaw) | Yes | Yes
+| 🧭 Metadata view (EXIF, map) | Yes | Yes
+| 🔎 Search by metadata, objects and image tags | Yes | No
+| ⚙️ Administrative functions (user management) | N/A | Yes
 
-### Mobile
-<p align="left">
-  <img src="design/login-screen.png" width="150" title="Login With Custom URL">
-  <img src="design/backup-screen.png" width="150" title="Backup Setting Info">
-  <img src="design/selective-backup-screen.png" width="150" title="Backup Setting Info">
-  <img src="design/home-screen.jpeg" width="150" title="Home Screen">
-  <img src="design/search-screen.jpeg" width="150" title="Curated Search Info">
-  <img src="design/shared-albums.png" width="150" title="Shared Albums">
-  <img src="design/nsc6.png" width="150" title="EXIF Info">
-</p>
 
-### Web
-<p align="left">
-  <img src="design/web-home.jpeg"  width="49%" title="Home Dashboard">
-  <img src="design/web-detail.jpeg" width="49%" title="Detail">
-</p>
-
-# Note
+  <br/>  
 
-**!! NOT READY FOR PRODUCTION! DO NOT USE TO STORE YOUR ASSETS !!**
+# Screenshots
 
-This project is under heavy development, there will be continuous functions, features and api changes.
+### Mobile
+| | | | | |
+| - | - | - | - | - |
+| <img src="design/login-screen.png" width="150" title="Login With Custom URL"> <p align="center"> Login with custom URL </p> | <img src="design/backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup Settings </p> | <img src="design/selective-backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup selection </p> | <img src="design/home-screen.jpeg" width="150" title="Home Screen"> <p align="center"> Home Screen </p> | <img src="design/search-screen.jpeg" width="150" title="Curated Search Info"> <p align="center"> Curated search </p> |
+| <img src="design/shared-albums.png" width="150" title="Shared Albums"> <p align="center"> Shared albums </p>  |  <img src="design/nsc6.png" width="150" title="EXIF Info"> <p align="center"> EXIF info </p>  | <img src="https://media.giphy.com/media/y8ZeaAigGmNvlSoKhU/giphy.gif" width="150" title="Loading ~4000 images/videos"> <p align="center"> Loading ~4000 images/videos </p>  |
```

Comment:
![image](https://user-images.githubusercontent.com/27055614/185256355-716f2b96-72b8-4273-86cb-316a81898833.png)
I think the table format here makes the layout kinda hard to follow as the indentation offset for each cell

Reply:
let's try a different ideas of display these messages or we can keep it linearly

Reply:
I just changed it to a linear display with centered images. Maybe you think this is better.

Also removed the `.env` example text and replaced it with a link (that opens in a new tab) to the `.env.example` file that is present in the repo. This way you don't need to change the `.env` in two places and users are linked directly to the file itself. Also this optimizes the space on the readme so steps look more simple at a glance.